Basic-Scripts/Rock_paper_scissor.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Aug 17 14:24:23 2020
+
+@author: Gulshan
+"""
+# Function rock_paper_scissor--->
+def rock_paper_scissor(num1,num2,bit1,bit2):
+    p1=int(num1[bit1])%3

```

Comment:
@gulshanbaraik01 Use meaningful and readable names instead of p1 p2

Reply:
Hey, I updated it as per your recent feedback. Kindly check and add it.